test/libsolidity/semanticTests/storageLayoutSpecifier/inline_assembly_direct_store.sol

```diff
@@ -0,0 +1,14 @@
+contract C layout at 42 {
+    uint x;
+    function f() public returns (uint) {
+        assembly {
+            sstore(42, 16)
+        }
+        return x;
+    }
+}
+// ====
+// EVMVersion: >=prague
+// compileViaYul: true
```

Comment:
Any reason all the new semantic tests are limited to Prague? They don't seem to be using any features not available on Cancun or even older EVMs.

Also, even if the feature is only implemented in the IR codegen, we should still have at least some of basic ones set to legacy and expecting the `UnimplementedFeatureError`.

Reply:
Only the transient storage test needs Cancun as evm version. I think I forgot to remove this and copied it all over.
> Also, even if the feature is only implemented in the IR codegen, we should still have at least some of the basic tests set to legacy and expecting the UnimplementedFeatureError.

I just discovered `compileViaYul: also` which is what I thought it was doing with the value `true`. So now I guess it is running both IR and legacy codegen, right?

Reply:
Yes. `false` runs only legacy, `true` runs only IR, `also` runs both. But `also` is the default, so you can omit it.